flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -538,6 +550,11 @@ under the License.
 							<goal>shade</goal>
 						</goals>
 						<configuration>
+							<artifactSet>
+								<excludes>
+									<exclude>org.apache.flink:flink-connector-kafka_${scala.binary.version}</exclude>
```

Comment:
Maybe this one should exclude `0.8`, `0.9` and `0.11` as well for the sake of  consistency?

Reply:
Ok, right. We can only exclude the modern one and `0.11`? And we can still exclude everything from "modern" `KafkaExample`?

Reply:
yes.